app/models/event_instance.rb

```diff
@@ -1,4 +1,5 @@
 class EventInstance < ActiveRecord::Base
+  after_create :create_related_event
```

Comment:
the pain of these callbacks usually comes when they are triggered unexpectedly - I've had enough of that pain to want to avoid them :-)

I'd prefer to have a service object codify the business logic relationships between models and have that service object be called from the controller